configuration.yaml

```diff
@@ -0,0 +1,490 @@
+---
+variables:
+  XLA_IR_DEBUG:
+    description:
+      - Enables the Python stack trace to be captured where creating IR nodes,
+        hence allowing to understand which PyTorch operation was responsible
+        for generating the IR.
+    type: bool
+    default_value: false
+  XLA_HLO_DEBUG:
+    description:
+      - Enables the Python stack frame captured when XLA_IR_DEBUG is active, to
+        be propagated to the XLA HLO metadata.
+    type: bool
+    default_value: false
+  XLA_GET_TENSORS_OPBYOP:
+    description:
+      - Enables pure OpByOp dispatch. The PyTorch/XLA software tries to fuse
+        together many PyTorch operations into a single computation graph, but
+        sometimes, either for debugging, or in case the PyTorch code have a
+        very dynamic nature (in shapes or graph terms), it is better to force
+        the execution in OpByOp mode (every IR node is lowered into a separate
+        XLA computation, and chain-executed). This environment variable, if set
+        to true, enables OpByOp during the "get tensors" operation (the
+        operation used by PyTorch/XLA to fetch intermediate values back from
+        the TPU device into PyTorch CPU tensors).
+    type: bool
+    default_value: false
+  XLA_SYNC_TENSORS_OPBYOP:
+    description:
+      - The same as XLA_GET_TENSORS_OPBYOP but for "sync tensors" operation(the
+        operation used at the end of a step, to flush pending IR computations
+        and materialize them into TPU device data).
+    type: bool
+    default_value: false
+  XLA_SYNC_WAIT:
+    description:
+      - Forces the XLA tensor sync operation to wait for its completion, before
+        moving to the next step.
+    type: bool
+    default_value: false
+  TF_CPP_LOG_THREAD_ID:
+    description:
+      - If set to true, the TF logs will show the thread ID helping with debugging
+        multithreaded processes.
+    type: bool
+    default_value: false
+  XLA_DUMP_HLO_GRAPH:
+    description:
+      - If set to true in case of a compilation or execution error the
+        offending HLO graph will be dumped as part of the runtime error raised
+        by xla_util.cc.
+    type: bool
+    default_value: false
+  XLA_TEST_DUMP_METRICS:
+    description:
+      - Controls whether or not metrics are dumped in test tear down.
+    type: bool
+    default_value: false
+  XLA_NO_SPECIAL_SCALARS:
+    description:
+      - When set to false, this will enable a compiler optimization to route
```

Comment:
it is not really a compiler optimization, it is more of we decided to pass scale value 0 and 1 as constant rather than device data to enable some compiler optimization.

Reply:
Done